src/directives/measureazimut.js

```diff
@@ -0,0 +1,76 @@
+goog.provide('ngeo.measureazimutDirective');
+
+goog.require('ngeo');
+goog.require('ngeo.interaction.MeasureAzimut');
+goog.require('ol.Feature');
+goog.require('ol.geom.Polygon');
+goog.require('ol.style.Style');
+
+
+/**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {gettext} gettext Gettext service.
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname ngeoDrawpoint
+ */
+ngeo.measureazimutDirective = function($compile, gettext) {
+  return {
+    restrict: 'A',
+    require: '^^ngeoDrawfeature',
+    /**
+     * @param {!angular.Scope} $scope Scope.
+     * @param {angular.JQLite} element Element.
+     * @param {angular.Attributes} attrs Attributes.
+     * @param {ngeo.DrawfeatureController} drawFeatureCtrl Controller.
+     */
+    link: function($scope, element, attrs, drawFeatureCtrl) {
+
+      var helpMsg = gettext('Click to start drawing azimut');
```

Comment:
I don't really get it, i know it comes from Lux.
What is `helpMsg` value, the translation of the key ?

Does `translate` directive below is usefull ? I think something is redundant here.